go/vt/vttablet/tabletmanager/rpc_replication_test.go

```diff
@@ -79,117 +74,3 @@ func TestPromoteReplicaReplicationManagerFailure(t *testing.T) {
 	// At the end we expect the replication manager to be stopped.
 	require.True(t, tm.replManager.ticks.Running())
 }
-
-func captureStderr(f func()) (string, error) {
-	old := os.Stderr // keep backup of the real stderr
-	r, w, err := os.Pipe()
-	if err != nil {
-		return "", err
-	}
-	os.Stderr = w
-
-	outC := make(chan string)
-	// copy the output in a separate goroutine so printing can't block indefinitely
-	go func() {
-		var buf bytes.Buffer
-		io.Copy(&buf, r)
-		outC <- buf.String()
-	}()
-
-	// calling function which stderr we are going to capture:
-	f()
-
-	// back to normal state
-	w.Close()
-	os.Stderr = old // restoring the real stderr
-	return <-outC, nil
-}
-
-func TestTabletManager_fixSemiSync(t *testing.T) {
```

Comment:
Do we have sufficient unit test coverage after removing this?

Reply:
This test was only checking the log messages printed out from the function `fixSemiSync` if the RPC setting and `enable_semi_sync` flag don't match. The test is irrelevant now, since there are no logs anymore.